wrappers/kratos_CFD_wrapper.py

```diff
@@ -400,18 +391,42 @@ def __importKratosConditions(self, src, dst, entitylist=None):
 
                 self.free_id += 1
 
-            condition_id = self.uuid_to_id_condition_map[condition.uid]
+                condition_id = self.uuid_to_id_condition_map[condition.uid]
+
+                dst.CreateNewCondition(
+                    "WallCondition3D",
+                    condition_id,
+                    [self.uuid_to_id_node_map[p] for p in condition.points],
+                    self.element_properties)
+
+    def importKratosDof(self, src, dst):
+        """ Fix the Dof for the appropiate nodes
 
-            dst.CreateNewCondition(
-                "WallCondition3D",
-                condition_id,
-                [self.uuid_to_id_node_map[p] for p in condition.points],
-                self.element_properties)
+        Iterates over all nodes in the kratos mesh and fix its DoF
+        according with the information of the SimPhoNy mesh.
+
+        """
+
+        # for point in src.iter_points():
+
+        #     node_id = self.uuid_to_id_node_map[point.uid]
+
+        #     node = dst.GetNodes()[node_id]
+
+        #     if point.data[CUBAExtension.IMPOSED_VEL] == 1:
+        #         node.Fix(VELOCITY_X)
+        #         node.Fix(VELOCITY_Y)
+        #         node.Fix(VELOCITY_Z)
+
+        #     if point.data[CUBAExtension.IMPOSED_PRES] == 1:
+        #         node.Fix(PRESSURE)
+
+        pass
 
     # FileIO
 
     def read_modelpart(self, filename):
-        """ Reads a Kratos formated modelpart NYI
+        """ Reads a Kratos formated modelpart
```

Comment:
Currently, the user needs to an ```read_modelpart``` where an existing Kratos ModelPart file is used to create a SimPhoNy mesh that can then be added to the wrapper.

Is there a description anywhere (or is it being planned) that describes the SimPhonyMesh that this wrapper needs.

Maybe an issue could be opened to add documentation about what the Mesh should be like (what CUBA should be used on cells, points etc..) for this kratos wrapper.